### ISSUE TITLE
Different closure formatting when it begins with loop vs while

### ISSUE THREAD

**Issue by stjepang** [rust-lang/rustfmt#3475]
Input:

```rust
fn main() {
    foo(|| loop { foo(); });
    foo(|| while true { foo(); });
    foo(|| if true { foo(); });
}
```

Output:

```rust
fn main() {
    foo(|| loop {
        foo();
    });
    foo(|| {
        while true {
            foo();
        }
    });
    foo(|| {
        if true {
            foo();
        }
    });
}
```

What I expected:

```rust
fn main() {
    foo(|| {
        loop {
            foo();
        }
    });
    foo(|| {
        while true {
            foo();
        }
    });
    foo(|| {
        if true {
            foo();
        }
    });
}
```

Version: `rustfmt 1.1.0-nightly (1427e4c 2019-03-17)`

**Comment by rchaser53** [comment]
This was resolved in #3334
Add config to rustfmt.toml like below.

```
version = "Two"
```

**Comment by stjepang** [comment]
Thanks!
